### PR TITLE
Feature/fgbs 001

### PR DESCRIPTION
**Pull request by MatthewWaanders**

### CODE REVIEW

File: drivers/FGBS-001/device.js [hunk 1]
```diff
@@ -0,0 +1,181 @@
+'use strict';
+
+const Homey = require('homey');
+const ZwaveDevice = require('homey-meshdriver').ZwaveDevice;
+
+class FibaroUniversalBinarySensor extends ZwaveDevice {
+
+	onMeshInit() {
+
+		/*
```
Comment (RobinBol) [inline drivers/FGBS-001/device.js]: Indentation

File: drivers/FGBS-001/device.js [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+		});
+
+		/*
+    	=========================================================================
+         Mapping alarm_generic capabilities to scene activation commands
+    	=========================================================================
+         */
+		this.registerCapability('alarm_generic.contact1', 'SCENE_ACTIVATION', {
+			report: 'SCENE_ACTIVATION_SET',
+			reportParser: (report) => this._alarmGenericReportParser(report, 1),
```
Comment (RobinBol) [inline drivers/FGBS-001/device.js]: Same as with the other device, what happens with reports if the command class version is higher?

File: drivers/FGBS-001/device.js [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+		});
+		this.registerCapability('alarm_generic.contact2', 'SCENE_ACTIVATION', {
+			report: 'SCENE_ACTIVATION_SET',
+			reportParser: (report) => this._alarmGenericReportParser(report, 2),
```
Comment (RobinBol) [inline drivers/FGBS-001/device.js]: Same here

**Comment by RobinBol** [comment]
What are you planning on doing with the open comments?

**Comment by MatthewWaanders** [comment]
As discussed internally we decided it's more common to override all report parsers when a developer uses `reportParser`. So I'm going to implement that in MeshDriver, until then this is not solvable in a proper way.

File: drivers/FGBS-001/device.js (hunk 2 above)
Comment (MatthewWaanders) [reply drivers/FGBS-001/device.js]: Fixed, not auto closed because it's fixed on a new line

**Pull request merged**